tests/info_extraction/InfoExtractorTest.py

```diff
@@ -46,7 +46,7 @@ def find_information(self, fname, parsed_file):
 
         class NoInfoExtractor(InfoExtractor):
 
-            def parse_file(self, file_content):
+            def parse_file(self, fname, file_content):
                 return file_content
```

Comment:
Can you add an extractor that just returns `fname` (instead of `file_content` since you added this new param) so that we can sleep peacefully in the night? :P